Networking/NetworkingTests/Remote/CouponsRemoteTests.swift

```diff
@@ -291,6 +291,57 @@ final class CouponsRemoteTests: XCTestCase {
         }
         XCTAssertEqual(resultError, .unacceptableStatusCode(statusCode: 500))
     }
+
+    // MARK: - Load coupon report
+
+    /// Verifies that loadCouponReport properly parses the `CouponReport` sample response.
+    ///
+    func test_loadCouponReport_properly_returns_parsed_report() {
+        // Given
+        let remote = CouponsRemote(network: network)
+        network.simulateResponse(requestUrlSuffix: "reports/coupons", filename: "coupon-reports")
+
+        // When
+        let result = waitFor { promise in
+            remote.loadCouponReport(for: self.sampleSiteID, couponID: 571) { (result) in
+                promise(result)
+            }
+        }
+
+        // Then
+        XCTAssert(result.isSuccess)
+        guard let returnedReport = try? result.get() else {
+            XCTFail("Expected parsed CouponReport not found in response")
+            return
+        }
```

Comment:
nit: can also simplify this a little by the following and marking the test case as `throws`
```suggestion
        let returnedReport = try XCTUnwrap(result.get())
```

Reply:
thanks for updating other places in this file! 💟